src/qibo/ui/mpldrawer.py

```diff
@@ -610,13 +593,13 @@ def _process_gates(array_gates):
             item += (init_label,)
 
             for qbit in gate._target_qubits:
-                if qbit is tuple:
+                if type(qbit) is tuple:
                     item += ("q_" + str(qbit[0]),)
                 else:
                     item += ("q_" + str(qbit),)
 
             for qbit in gate._control_qubits:
-                if qbit is tuple:
+                if type(qbit) is tuple:
                     item += ("q_" + str(qbit[0]),)
                 else:
                     item += ("q_" + str(qbit),)
```

Comment:
```suggestion
                item_add = ("q_" + str(qbit[0]),) if isinstance(qbit, tuple) else  ("q_" + str(qbit),)
                item += item_add
```

Reply:
thanks @renatomello , As this is an enhance and not a fix I add this in https://github.com/qiboteam/qibo/pull/1438

Reply:
As you wish @sergiomtzlosa, we can also just accept this suggestion here and keep the other one to fix the coverage :) Let me know.

Reply:
ok @MatteoRobbiati just accept the fix, it is fine for me 👍